### PR TITLE
Onehanded cigs, and a bunch of other stuff

### PR DESCRIPTION
**Pull request by alsoandanswer**
Surprisingly, doesn't really change *that* much user-facing side.

For some stupid reason, cigar cases were using a rehash of cigarette packet code for no good reason. So I just made it a parent item, and it works just fine.

also added a tooltip to share the rather obscure feature of putting a cigarette in your mouth.

resprites the tajaran cigarette packets to more accurately match where they come from

dreammaker_p9iUbnoyfP

Also fixes the bug with cheap lighters not updating when lit

And adds a less grating sound to opening stuff (that previously used paper pickup sound)

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!bugfix

**Comment by alsoandanswer** [comment]
!review

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 1]
```diff
@@ -152,8 +152,9 @@
 //CIG PACK//
 ////////////
 /obj/item/storage/fancy/cigarettes
-	name = "Trans-Stellar Duty Free cigarette packet"
+	name = "Trans-Stellar Duty Frees cigarette packet"
```
Comment (Doxxmedearly) [inline code/game/objects/items/weapons/storage/fancy.dm:155]:
```suggestion
	name = "Trans-Stellar Duty Free cigarette packet"
```
Duty Free is the accurate term

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 2]
```diff
@@ -220,120 +221,52 @@
...
-	desc = "A packet of six imported DromedaryCo cancer sticks. A label on the packaging reads, \"Wouldn't a slow death make a change?\""
-	icon_state = "Dpacket"
-	item_state = "Dpacket"
-	cigarette_to_spawn = /obj/item/clothing/mask/smokable/cigarette/dromedaryco
-
-/obj/item/storage/fancy/cigar
+/obj/item/storage/fancy/cigarettes/cigar
 	name = "cigar case"
-	desc = "A case for holding your cigars when you are not smoking them."
+	desc = "A luxurious tote for your fat tokes."
```
Comment (Doxxmedearly) [inline code/game/objects/items/weapons/storage/fancy.dm:238]: Cringe

Comment (alsoandanswer) [reply code/game/objects/items/weapons/storage/fancy.dm:238]: it rhymes, also _marginally_ better than
name : "cigar case"
desc : "case for holding cigars" which is basically the same thing but more verbose

File: code/game/objects/items/weapons/storage/fancy.dm (hunk 1 above)
Comment (alsoandanswer) [reply code/game/objects/items/weapons/storage/fancy.dm:155]: it's the name of the cigarettes, not the packet

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 3]
```diff
@@ -211,7 +212,7 @@
 		reagents.trans_to_obj(W, (reagents.total_volume/contents.len))
 		user.equip_to_slot_if_possible(W, slot_wear_mask)
 		reagents.maximum_volume = 15 * contents.len
-		to_chat(user, "<span class='notice'>You take a cigarette out of the pack.</span>")
+		user.visible_message(SPAN_NOTICE("[user] casually pulls out a [cigarette_to_spawn] from \the [src] with their mouth."))
```
Comment (Geevies) [inline code/game/objects/items/weapons/storage/fancy.dm:215]:
```suggestion
		user.visible_message("<b>[user]</b> casually pulls out a [cigarette_to_spawn] from \the [src] with their mouth.", range = 3)
```

**Pull request merged**